trk/src/Screens/TabScreens/Record/Backend/logic.js

```diff
@@ -26,6 +28,42 @@ export const useHomeScreenLogic = (props) => {
 
     const dispatch = useDispatch();
 
+    //Next PR these will actually be calculated rather than hardcoded (set the true value to the option with the most messages in JSON file)
+    let isFirstTimeUser = false;
+    let isFTUNotFirstClimb = false;
+    let isReturningUserFirstClimbOfDay = false;
+    let isReturningUserFirstClimbOfAnotherSession = true;
+    let isReturningUserNotFirstClimb = false;
+
```

Comment:
hardcoded for now, will be calculate din next PR

Reply:
great start 👍